tools/package-lock.json

```diff
@@ -0,0 +1,754 @@
+{
+  "name": "template-schema-validation-tools",
```

Comment:
Sure. Just following the directions from the tool:

```
[D:\git\repos\azure-resource-manager-schemas\tools]-> npm install
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated to-iso-string@0.0.2: to-iso-string has been deprecated, use @segment/to-iso-string instead.
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue

> spawn-sync@1.0.15 postinstall D:\git\repos\azure-resource-manager-schemas\tools\node_modules\spawn-sync
> node postinstall

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN template-schema-validation-tools@1.0.0 No repository field.
npm WARN The package mocha is included as both a dev and production dependency.

added 98 packages from 436 contributors in 12.067s
[D:\git\repos\azure-resource-manager-schemas\tools]->
```

Note: **You should commit this file.** in the above messages.

Reply:
Done.